src/rpc/server.cpp

```diff
@@ -399,7 +399,10 @@ static inline JSONRPCRequest transformNamedArguments(const JSONRPCRequest& in, c
     const std::vector<UniValue>& values = in.params.getValues();
     std::unordered_map<std::string, const UniValue*> argsIn;
     for (size_t i=0; i<keys.size(); ++i) {
-        argsIn[keys[i]] = &values[i];
+        auto inserted = argsIn.emplace(keys[i], &values[i]);
+        if (!inserted.second) {
```

Comment:
nit: I think it's reasonable enough to skip using the intermediate `inserted` variable entirely, but if you think it's better for readability I'd suggest going all the way and make `inserted` the actual boolean directly?
```suggestion
        auto [_, inserted] = argsIn.emplace(keys[i], &values[i]);
        if (!inserted) {
```